src/sass/_forms.scss

```diff
@@ -140,6 +140,10 @@
 }
 
 .image-select-modal {
+  .p-modal__dialog {
+    overflow-y: hidden;
```

Comment:
Half of a filter option (i.e. its label) might be visible and with this there is no way to get to the dropdown to use the filter. Shall we remove this bit?

Reply:
This is the problem that style is trying to solve:

![image](https://github.com/canonical/lxd-ui/assets/56583786/6ef068d3-4ca0-4801-a5aa-87924d9e5ee0)

So, at around ~590px height, we have double scrollbars, on the table and on the modal. I applied that style to avoid that, because the one on the modal is unnecessary and can be hidden. I cannot reproduce/don't understand the issue that you mentioned.